.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,28 @@
+name: PR Checks
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+  issues:
+    types:
+      - milestoned
+      - demilestoned
```

Comment:
There's a current limitation with GitHub webhooks and it's that milestoned/demilestoned actions only triggers for issues and not pull requests. It still works for pull requests, but required some more work and it triggers for any issue (issues and pull requests).